azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/service/sdk/ChatSDKWrapper.kt

```diff
@@ -73,6 +74,17 @@ internal class ChatSDKWrapper(
         return future
     }
 
+    override fun startEventNotifications(context: Context) {
+        if (startedEventNotifications) return
+        chatAsyncClient.startRealtimeNotifications(context) {}
+        startedEventNotifications = true
```

Comment:
Does startRealTimeNotification have any success callback? I mean can we set true or false within that scope given startRealtimeNotifications(context) {
 // scope
}

Reply:
startEventNotifications() now takes an error handler function as an argument and this will be called with the exception if it happens in https://github.com/Azure/communication-ui-library-android/commit/093c6e208e9d4ce907bb469abfc93e324bc083e8